models/models.go

```diff
@@ -174,16 +174,16 @@ func (pr *ParseRequest) ValidateParseRequest() error {
 	switch units := pr.CellSizeUnits; units {
 	case "%":
 		if pr.CurrentTableWidth <= 0 {
-			log.InfoC(pr.Filename, "size_units is '%' but current_table_width is not specified - cannot convert from px", nil)
+			log.Event(nil, fmt.Sprintf("%s: size_units is 'percentage', but current_table_width is not specified - cannot convert from px", pr.Filename), log.INFO)
```

Comment:
the new logging guideline is that we should provide a concise message here, and then provide the values as part of log.Data, it should be something like:
```
log.Event(ctx, "size_units is 'percentage' but current_table_width is not specified -cannot convert from px", log.Data{"file_name": pr.Filename}
```

Reply:
Updated all logs to match our convention